### PR TITLE
guide for aws lambda dotnet

### PR DESCRIPTION
**Pull request by bruno-garcia**

### CODE REVIEW

File: src/platforms/dotnet/guides/aws-lambda/index.mdx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+public class LambdaEntryPoint : Amazon.Lambda.AspNetCoreServer.APIGatewayProxyFunction
+{
+    protected override void Init(IWebHostBuilder builder)
+    {
+        builder
+            // Add Sentry
+            .UseSentry(o =>
+            {
+              o.Dsn = "__PUBLIC_DSN__;
```
Comment (bruno-garcia) [inline src/platforms/dotnet/guides/aws-lambda/index.mdx:42]:
```suggestion
              o.Dsn = "__PUBLIC_DSN__";
```

File: src/platforms/dotnet/guides/aws-lambda/index.mdx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+            .UseStartup<Startup>();
+    }
+}
+```
+
+Check out the [Sentry ASP.NET Core](/aspnetcore) documentation for the complete set of options.
+
+## Samples
+
+- A [samples on GitHub](https://github.com/getsentry/sentry-dotnet/tree/main/samples/Sentry.Samples.Aws.Lambda.AspNetCoreServer) demonstrates Sentry on AWS Lambda. (**C#**)
```
Comment (PeloWriter) [inline src/platforms/dotnet/guides/aws-lambda/index.mdx:55]:
```suggestion
- Our [samples on GitHub](https://github.com/getsentry/sentry-dotnet/tree/main/samples/Sentry.Samples.Aws.Lambda.AspNetCoreServer) demonstrate Sentry on AWS Lambda. (**C#**)
```

**Review by PeloWriter — APPROVED** [review]
minor nit. Otherwise, looks good.

**Comment by bruno-garcia** [comment]
Fixed. Thanks @PeloWriter

File: src/platforms/dotnet/guides/aws-lambda/config.yml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+title: AWS Lambda
+sdk: sentry.dotnet.aspnetcore
```
Comment (bruno-garcia) [inline src/platforms/dotnet/guides/aws-lambda/config.yml:2]: @PeloWriter I don't follow, the sdk name is `sentry.dotnet.aspnetcore`

File: src/platforms/dotnet/guides/aws-lambda/index.mdx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+              o.Dsn = "__PUBLIC_DSN__";
+              // Required in Serverless environments
+              o.FlushOnCompletedRequest = true;
+            })
+            .UseStartup<Startup>();
+    }
+}
+```
+
+Check out the [Sentry ASP.NET Core](/aspnetcore) documentation for the complete set of options.
```
Comment (bruno-garcia) [inline src/platforms/dotnet/guides/aws-lambda/index.mdx:49]:
```suggestion
Check out the [Sentry ASP.NET Core](aspnetcore) documentation for the complete set of options.
```

File: src/platforms/dotnet/guides/aws-lambda/index.mdx [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+              o.Dsn = "__PUBLIC_DSN__";
+              // Required in Serverless environments
+              o.FlushOnCompletedRequest = true;
+            })
+            .UseStartup<Startup>();
+    }
+}
+```
+
+Check out the [Sentry ASP.NET Core](aspnetcore) documentation for the complete set of options.
```
Comment (bruno-garcia) [inline src/platforms/dotnet/guides/aws-lambda/index.mdx:49]:
```suggestion
Check out the [Sentry ASP.NET Core](/platforms/dotnet/guides/aspnetcore/) documentation for the complete set of options.
```

File: src/platforms/dotnet/guides/aws-lambda/index.mdx [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+              o.Dsn = "__PUBLIC_DSN__";
+              // Required in Serverless environments
+              o.FlushOnCompletedRequest = true;
+            })
+            .UseStartup<Startup>();
+    }
+}
+```
+
+Check out the [Sentry ASP.NET Core](/platforms/dotnet/guides/aspnetcore/) documentation for the complete set of options.
```
Comment (bruno-garcia) [inline src/platforms/dotnet/guides/aws-lambda/index.mdx:49]:
```suggestion
Check out the <PlatformLink to="/platforms/dotnet/guides/aspnetcore/">ASP.NET Core SDK</PlatformLink> documentation for the complete list of options.
```

File: src/platforms/dotnet/guides/aws-lambda/index.mdx [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+              o.Dsn = "__PUBLIC_DSN__";
+              // Required in Serverless environments
+              o.FlushOnCompletedRequest = true;
+            })
+            .UseStartup<Startup>();
+    }
+}
+```
+
+Check out the <PlatformLink to="/platforms/dotnet/guides/aspnetcore/">ASP.NET Core SDK</PlatformLink> documentation for the complete list of options.
```
Comment (bruno-garcia) [inline src/platforms/dotnet/guides/aws-lambda/index.mdx:49]:
```suggestion
Check out the [Sentry ASP.NET Core](/platforms/dotnet/guides/aspnetcore/) documentation for the complete set of options.
```

**Pull request merged**